crowbar_framework/config/repos-ses.yml

```diff
@@ -37,27 +37,27 @@ suse-12.1:
       features: ["ha"]
       repomd:
         tag: "obsrepository://build.suse.de/SUSE:Updates:SLE-HA:12-SP1:x86_64/update"
-        key_md5: "9a62177e1c6852d48453ed3909956d6b"
+        fingerprint: "FEAB 5025 39D8 46DB 2C09 61CA 70AF 9E81 39DB 7C82"
       url:
       smt_path: "SUSE/Updates/SLE-HA/12-SP1/x86_64/update"
       ask_on_error: false
     suse-enterprise-storage-3-pool:
       name: "SUSE-Enterprise-Storage-3-Pool"
-      required: "optional"
+      required: "mandatory"
```

Comment:
Jut for clarification: It's now mandatory because it's the ```repos-ses.yml``` file which is used for deploying SES with crowbar, right?